src/com/itschool/session4/homework/ex1/AriaDreptunghiului.java

```diff
@@ -0,0 +1,23 @@
+package com.itschool.session4.homework.ex1;
+
+import java.util.Scanner;
+
+public class AriaDreptunghiului {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Introduceti lungimea dreptunghiului: ");
+        double lungimea = scanner.nextDouble();
+
+        System.out.println("Introduceti latimea dreptunghiului: ");
+        double latimea = scanner.nextDouble();
+
+        double aria = lungimea * latimea;
+
+        System.out.println("Aria este: " + aria);
+
+        scanner.close();

```

Comment:
can you explain what this line does and why do you need it?

Reply:
Scanner deschide o sursa de date pentru a putea sa le citetesca si este mai bine să eliberezi resursele, prin a inchide obiectul Scanner.

Reply:
asa e, dar aici nu este necesar pt ca noi terminam imediat programul dupa ce afisam diferite chestii si se inchid toate procesele, inclusiv resursa folosita de `Scanner`, in concluzie nu e necesara aceasta linie in situatia de fata (nici nu e gresit).
P.S: voiam sa ma asigur ca nu ai copiat cod pe care nu il intelegi :p